tests/at_end2end_test/test/notify_test.dart

```diff
@@ -66,5 +66,48 @@ void main() async {
     expect(notificationListJson[0]['from'], currentAtSign);
     expect(notificationListJson[0]['to'], sharedWithAtSign);
     expect(notificationListJson[0]['value'], isNotEmpty);
+    expect(notificationListJson[0]['value'] != value,
+        true); //encrypted value should be different from actual value
+  });
+  test('Notify a key with value by setting encryptValue to false', () async {
+    var uuid = Uuid();
+    // Generate  uuid
+    var randomValue = uuid.v4();
+    var phoneKey = AtKey()
+      ..key = 'phone$randomValue'
+      ..sharedWith = sharedWithAtSign
+      ..metadata = (Metadata()..ttr = 60000)
+      ..namespace = namespace;
+
+    // Appending a random number as a last number to generate a new phone number
+    // for each run.
+    var value = '+1 100 200 30';
+    // Setting currentAtSign atClient instance to context.
+    currentAtClientManager = await AtClientManager.getInstance()
+        .setCurrentAtSign(currentAtSign, namespace,
+            TestPreferences.getInstance().getPreference(currentAtSign));
+    final notificationResult = await currentAtClientManager
+        .atClient.notificationService
+        .notify(NotificationParams.forUpdate(phoneKey, value: value),
+            encryptValue: false);
+    expect(notificationResult, isNotNull);
+    expect(notificationResult.notificationStatusEnum,
+        NotificationStatusEnum.delivered);
+
+    // Setting sharedWithAtSign atClient instance to context.
+    await AtClientManager.getInstance().setCurrentAtSign(
+        sharedWithAtSign,
+        namespace,
+        TestPreferences.getInstance().getPreference(sharedWithAtSign));
+    var notificationListResult = await AtClientManager.getInstance()
+        .atClient
+        .notifyList(regex: 'phone$randomValue');
+    expect(notificationListResult, isNotEmpty);
+    notificationListResult = notificationListResult.replaceFirst('data:', '');
+    final notificationListJson = jsonDecode(notificationListResult);
+    print(notificationListJson);
+    expect(notificationListJson[0]['from'], currentAtSign);
+    expect(notificationListJson[0]['to'], sharedWithAtSign);
+    expect(notificationListJson[0]['value'], value);
```

Comment:
can you also add an expect for isEncrypted please?

Reply:
I am getting isEncrypted=true for this test.
[{id: 79a2d96e-ff34-442c-aac0-97a556fcbcfc, from: @ce2e1, to: @ce2e2, key: @ce2e2:phone2a9c23ff-cb4c-4d41-a50d-276d87d24820.e2e_test@ce2e1, value: +1 100 200 30, operation: update, epochMillis: 1726120153595, messageType: MessageType.key, isEncrypted: true, metadata: {encKeyName: null, encAlgo: null, ivNonce: null, skeEncKeyName: null, skeEncAlgo: null, availableAt: null, expiresAt: 2024-09-13 05:49:13.595Z***]
Is this because we don't have the changes on the server yet?